config/locales/en.yml

```diff
@@ -390,6 +390,23 @@ en:
       user: "User"
       liked: "Liked"
       action: "Action"
+  map:
```

Comment:
Hi, it looks like perhaps some extra changes were included here that aren't part of the core issue? Would it be possible to submit the PR with only the README changes? Thank you so much!

I hope this may be helpful!!

https://publiclab.org/wiki/contributing-to-public-lab-software#Rewinding+the+master+branch

In general there are more tips and guides on rebasing here: https://publiclab.org/wiki/developers#Resources

Thanks!

Reply:
Thank you @jywarren I made sure the the branch was rewind and updated the pull request